data/phi1348/abo014/__cts__.xml

```diff
@@ -1,11 +1,12 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi1348.abo014" xml:lang="lat" groupUrn="urn:cts:latinLit:phi1348">
-            <ti:title xml:lang="lat">Caligula</ti:title>
+            <ti:title xml:lang="eng">Caligula</ti:title>
```

Comment:
as above, I don't think it makes a difference but Latin seems just as valid as English to me....